codespell_lib/data/dictionary.txt

```diff
@@ -19452,78 +19460,161 @@ gayity->gaiety
 gaysha->geisha
 gayshas->geishas
 geeneric->generic
+geenral->general
+geenrally->generally
 geenrate->generate
 geenrated->generated
 geenrates->generates
+geenrating->generating
 geenration->generation
 geenrational->generational
+geenrations->generations
+geenrator->generator
+geenrators->generators
+geenric->generic
+geenrics->generics
 geeoteen->guillotine
 geeral->general
 gemetrical->geometrical
 gemetry->geometry
 gemoetry->geometry
 gemometric->geometric
+genaral->general
+genarally->generally
 genarate->generate
 genarated->generated
 genarates->generates
 genarating->generating
 genaration->generation
+genarations->generations
+genarator->generator
+genarators->generators
+genaric->generic
+genarics->generics
 genearal->general
 genearally->generally
+genearate->generate
+genearated->generated
+genearates->generates
+genearating->generating
+genearation->generation
+genearations->generations
+genearator->generator
+genearators->generators
+genearic->generic
+genearics->generics
+genearl->general
+genearlly->generally
+genearte->generate
 genearted->generated
+geneartes->generates
+genearting->generating
+geneartion->generation
+geneartions->generations
+geneartor->generator
+geneartors->generators
 geneate->generate
 geneated->generated
 geneates->generates
 geneating->generating
 geneation->generation
+geneations->generations
+geneator->generator
+geneators->generators
 geneic->generic, genetic,
 geneological->genealogical
 geneologies->genealogies
 geneology->genealogy
 generaates->generates
+generae->generate
+generaed->generated
+generaes->generates
 generage->generate
 generaged->generated
 generages->generates
 generaging->generating
 generaing->generating
+generaion->generation
+generaions->generations
 generall->generally, general,
 generaly->generally
 generalyl->generally
 generalyse->generalise
+generaor->generator
+generaors->generators
 generat->generate, general,
+generatd->generated
 generater->generator
 generaters->generators
 generatig->generating
 generatng->generating
 generaton->generation, generator,
+generatons->generations, generators,
+generatr->generator, generate,
+generatrs->generators, generates,
+generats->generates
+generatted->generated
+generattes->generates
 generatting->generating
+generattion->generation
+generattions->generations
+generattor->generator
+generattors->generators
+genereal->general
+genereally->generally
 genereate->generate
 genereated->generated
 genereates->generates
 genereating->generating
+genereation->generation
+genereations->generations
+genereator->generator
+genereators->generators
 genered->generated
```

Comment:
Just a suggestion, I'm letting you decide:
```
genered->generated, gendered,
```

Reply:
I found a few examples of `genered->gendered` and `genered->gender`, so I added both.